localhost

```diff
@@ -1 +1 @@
-localhost
+localhost ansible_python_interpreter=/usr/local/bin/python2.7
```

Comment:
That might help.  I'm not sure if that's why this wasn't working, or if ansible-pull's git checkout step doesn't use `group_vars/all`